subsys/logging/Kconfig

```diff
@@ -209,11 +209,37 @@ config LOG_INPLACE_PROCESS
 	bool "Enable in place processing"
 	help
 	  When enabled log is processed in the context of the call. It impacts
-	  performance of the system since time consuming operations are performed in
-	  the context of the log entry (e.g. high priority interrupt). When enabled
-	  LOG_BUFFER_SIZE can be reduced. Logger backends must support exclusive
-	  access to work flawlessly in that mode because one log operation can be
-	  interrupted by another one in higher priority context.
+	  performance of the system since time consuming operations are
+	  performed in the context of the log entry (e.g. high priority
+	  interrupt). When enabled LOG_BUFFER_SIZE can be reduced. Logger
+	  backends must support exclusive access to work flawlessly in that mode
+	  because one log operation can be interrupted by another one in higher
+	  priority context.
+
+config LOG_PROCESS_TRIGGER_THR
```

Comment:
THR was supposed to mean threshold. I've extended it to threshold to avoid confusion and modified prompt to "Amount of buffered logs which triggers processing thread."